openvpn/client.go

```diff
@@ -39,13 +39,17 @@ type openVpnClient struct {
 
 // NewClient creates openvpn client with given config params
 func NewClient(openvpnBinary string, config *ClientConfig, directoryRuntime string, middlewares ...management.Middleware) *openVpnClient {
+	managementAddress, err := management.GetAvailableAddress()
+	if err != nil {
+		return nil
```

Comment:
How this 'nil' should be handled in caller? Since its factory, all logic should be performed outside and result passed as argument, such as: 
func NewClient(managementAddress *Address ... ) *openVpnClient

Reply:
Good point! I'm on it:) Thanks!